src/components/Form.vue

```diff
@@ -38,6 +38,10 @@ import HeaderFormComponent from 'components/FormHeader.vue';
 import G3wFormFooter from 'components/FormFooter.vue';
 
 export default {
+
+  /** @since 3.8.6 */
+  name: 'form',
```

Comment:
I get the following while entering editing mode:

```log
[Vue warn]: Do not use built-in or reserved HTML elements as component id: form
```

Reply:
@Raruto ok. i'll change with 'g3w-form'